includes/features/Drupal-Core/User/adminstrator_role.feature

```diff
@@ -0,0 +1,27 @@
+Feature: module
```

Comment:
Leave or move back to product specific directory for now. We will re-evaluate a shared set of tests for later.